scripts/get-hopr-address.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# prevent sourcing of this script, only allow execution
+$(return >/dev/null 2>&1)
+test "$?" -eq "0" && { echo "This script should only be executed." >&2; exit 1; }
+
+# exit on errors, undefined variables, ensure errors in pipes are not hidden
+set -Eeuo pipefail
+
+declare mydir
+mydir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
+source "${mydir}/utils.sh"
+
+# $1 = optional: endpoint, defaults to http://localhost:3001
+declare endpoint="${1:-localhost:3001}"
```

Comment:
Please validate that the command line argument are present. And if not either fail, or emit a message saying we're defaulting to `localhost`